backend/src/bundles/files/helpers/get-unique-name.helper.ts

```diff
@@ -0,0 +1,11 @@
+import crypto from 'node:crypto';
+
+const FIRST_RANDOM_NUMBER = 3;
+const SECOND_RANDOM_NUMBER = 4;
+
+const getUniqueName = (fileName: string): string =>
+    `${fileName}-${crypto
+        .randomBytes(FIRST_RANDOM_NUMBER * SECOND_RANDOM_NUMBER)
+        .toString('base64')}`;
+
+export { getUniqueName };
```

Comment:
Can we use generateRadomId function to get unique name for every file ?

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/90120572/9c4f9c22-a94e-4416-ab21-1ce55b602f89)

I tried this and it generates cleaner file names.

![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/90120572/76c27346-1572-45d5-888a-53d3092da42d)